tools/unlabel

```diff
@@ -0,0 +1,28 @@
+#! /bin/sh
+#**************************************************************************
+#*                                                                        *
+#*                                 OCaml                                  *
+#*                                                                        *
+#*                            John Whitington                             *
+#*                                                                        *
+#*   Copyright 2020 Institut National de Recherche en Informatique et     *
+#*     en automatique                                                     *
+#*                                                                        *
+#*   All rights reserved.  This file is distributed under the terms of    *
+#*   the GNU Lesser General Public License version 2.1, with the          *
+#*   special exception on linking described in the file LICENSE.          *
+#*                                                                        *
+#**************************************************************************
+
+#Removes a label, i.e a space, a variable name, followed by a colon followed by
+#an alphabetic character or ( or '. This should avoid altering the contents of
+#comments.
+
+perl -p -e "s/ [a-z_]+:([a-z\('])/ \1/g" \
```

Comment:
Would it be possible to use `sed` instead of `perl` ? `sed` is already used in a number of places, contrary to `perl` which is not used anywhere in the build system.

Reply:
We're not in the build system (yet). As an example, `tools/ocamlsize` is in perl.

It requires a sed script which operates on OS X and GNU systems - this requires testing the uname and so on. I'm sure it could be done, and would be appropriate if we move this into the build system at some point.

Reply:
Fair enough, let's leave this for later.